gta/altv/egg-alt--v.json

```diff
@@ -1,40 +1,45 @@
+            "script": "#!\/bin\/bash\r\n# AltV Install Script\r\n#\r\n# Server Files: \/mnt\/server\r\n# BUILD=stable ## this should be an egg variable.\r\napt update && apt install -y libatomic1 zip unzip jq curl wget libc-bin && cd \/mnt\/server && wget -O 'update.sh' 'https:\/\/raw.githubusercontent.com\/Lhoerion\/altv-serverupdater\/master\/update.sh' && chmod +x .\/update.sh && .\/update.sh && chmod +x .\/altv-server\r\necho -e \"install complete\"",
```

Comment:
I will not accept you pulling another script to run the install. Users can run reinstall in 1.2+ so they can update that way.